README.md

```diff
@@ -7,36 +7,33 @@
 ![Python 3.12](https://img.shields.io/badge/Python-3.12-blue.svg)
 [![Docker Image Version (latest semver)](https://img.shields.io/docker/v/safeglobal/safe-cli?label=Docker&sort=semver)](https://hub.docker.com/r/safeglobal/safe-cli)
 
-# Safe-CLI
+# Safe CLI
 
-Command line utility for **Safe** contracts. Use it to manage your **Safe** easily from the command line.
+Safe CLI is a command-line utility for Safe contracts. You can use it to manage your Safe Smart Account from the command line.
```

Comment:
I see "The Safe Smart Account" as the protocol, and "Safe account" as a user account.

```suggestion
Safe CLI is a command-line utility for Safe contracts. You can use it to manage your Safe account from the command line.
```